cmd/kubelet/app/server.go

```diff
@@ -223,7 +223,7 @@ func NewKubeletServer() *KubeletServer {
 		RktPath:                        "",
 		RktStage1Image:                 "",
 		RootDirectory:                  defaultRootDir,
-		SerializeImagePulls:            true,
+		SerializeImagePulls:            util.BoolFlag{false, true},
```

Comment:
It'd be a bit clearer as well, especially since you're not using the field-name-less version of the struct initializer.